cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/HealthChecksIT.java

```diff
@@ -0,0 +1,70 @@
+package com.yugabyte.cdcsdk.testing;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.json.JSONObject;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.yugabyte.cdcsdk.testing.util.CdcsdkTestBase;
+import com.yugabyte.cdcsdk.testing.util.UtilStrings;
+
+public class HealthChecksIT extends CdcsdkTestBase {
+    private int healthCheckMappedPort;
+
+    @BeforeAll
+    public static void beforeAll() throws Exception {
+        initializeContainers();
+
+        // We only need the Kafka container to be started here as the CDCSDK server will try to
+        // establish a connection to it
+        kafkaContainer.start();
```

Comment:
No need to change this test. In the future for similar types of tests, NullConsumer can be used to simplify setup

Reply:
Definitely, will add the infra for null consumer in a separate PR.